README.md

```diff
@@ -1,28 +1,42 @@
+
+
+# iOS 3D Secure SDK
 
 The Checkout.com 3D Secure (3DS) mobile SDK allows you to provide a native 3DS2 experience in your mobile app, with visual styling that you can control. 
 
 The SDK handles the device data collection, communication with the card issuer, and presentation of 3D Secure challenges to the customer when required. 
 
+- [Features](#features)
+- [Minimum Requirements](#minimum-requirements)
+- [Installation](#installation)
+  - [CocoaPods](#cocoapods)
+- [Integration](#integration)
+  - [Checkout.com's 3DS server](#checkoutcoms-3ds-server)
+  - [Any 3DS provider](#any-3ds-provider)
+- [Payment Authorisation](#payment-authorisation)
+- [Dependencies](#dependencies)
+- [Help and Feedback](#help-and-feedback)
+- [License](#license)
+  
 👉&nbsp;&nbsp;[See the Integration Guide with Code Examples](https://www.checkout.com/docs/risk-management/3d-secure/sessions/non-hosted-sessions/3d-secure-mobile-sdks)
 
 📚&nbsp;&nbsp;[Read the reference documentation](https://checkout.github.io/checkout-mobile-docs/checkout-3ds-sdk-ios/index.html)
 
+
 ## Features
-1. Supports **3D Secure protocols 2.1.0 and 2.2.0.**
-2. Presents **native 3DS2 challenge screens to the user, with styling that you can customise** to best fit your apps and branding.
-3. Optionally **falls back to 3DS1** when 3DS2 is not available, returning the same authentication result to your app as for 3DS2. (This option can be configured by Checkout.com for your account.)
-4. Supports a **range of local languages and accessibility needs**, and allows you to set your own string translations.
-5. Meets requirements from **EMVCo and the PCI Security Standards Council**, specifically set for 3DS SDKs, so you can be sure it is interoperable with card issuers and that your customers’ sensitive data stays secure.
+- Supports **3D Secure protocols 2.1.0 and 2.2.0.**
+- Presents **native 3DS2 challenge screens to the user, with styling that you can customise** to best fit your apps and branding.
+- Optionally **falls back to 3DS1** when 3DS2 is not available, returning the same authentication result to your app as for 3DS2. (This option can be configured by Checkout.com for your account.)
+- Supports a **range of local languages and accessibility needs**, and allows you to set your own string translations.
+- Meets requirements from **EMVCo and the PCI Security Standards Council**, specifically set for 3DS SDKs, so you can be sure it is interoperable with card issuers and that your customers’ sensitive data stays secure.
 
 ## Minimum Requirements
-The 3DS SDK for iOS requires Xcode 13.1 and above with Swift version 5.6.2 and above, and supports apps targeting iOS 12.0 and above. It also supports Objective-C. There are two ways to integrate our 3DS SDK.
+The 3DS SDK for iOS requires Xcode 13.1 and above with Swift version 5.6.2 and above, and supports apps targeting iOS 11.0 and above. It also supports Objective-C. There are two ways to integrate into our 3DS SDK.
```

Comment:
I think we shouldn't add `into` here. It changes the meaning in a way as if merchants integrate their apps into our SDK, rather the opposite.

Reply:
Personally, I don't `into` is necessary here for it to make grammatical sense